tests/UI/campaigns/functional/BO/00_login/01_passwordReminder.js

```diff
@@ -0,0 +1,273 @@
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_login_passwordReminder';
+
+let browserContext;
+let page;
+let numberOfEmployees = 0;
+// maildev
+let newMail;
+const {smtpServer, smtpPort} = global.maildevConfig;
+const resetPasswordSuccessText = 'Please, check your mailbox.';
+const testMailSubject = 'Test message -- Prestashop';
+const resetPasswordMailSubject = 'Your new password';
+
+// new employee datas
+const createEmployeeData = new EmployeeFaker({
+  defaultPage: 'Products',
+  language: 'English (English)',
+  permissionProfile: 'Salesman',
+});
+
+// init mailListener
+const mailListener = mailHelper.createMailListener();
+
+describe('BO Password reminder', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+    mailHelper.startListener(mailListener);
+    // Handle every new email
+    mailListener.on('new', (email) => {
+      newMail = email;
+    });
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+    mailHelper.stopListener(mailListener);
+  });
+
+  describe('Go to BO to setup the smtp parameters', async () => {
+    it('should login in BO', async function () {
+      await loginCommon.loginBO(this, page);
+    });
+
+    it('should go to email parameters page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToEmailSetupPageForSetupSmtpParams', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.advancedParametersLink,
+        dashboardPage.emailLink,
+      );
+
+      await emailPage.closeSfToolBar(page);
+
+      const pageTitle = await emailPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(emailPage.pageTitle);
+    });
+
+    it('should fill the smtp parameters form fields', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'fillSmtpParametersFormField', baseContext);
+
+      const alertSuccessMessage = await emailPage.setupSmtpParameters(
+        page,
+        smtpServer,
+        DefaultCustomer.email,
+        DefaultCustomer.password,
+        smtpPort,
+      );
+
+      await expect(alertSuccessMessage).to.contains(emailPage.successfulUpdateMessage);
+    });
+
+    it('should check successful message after sending test email', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'sendTestEmail', baseContext);
+
+      const textResult = await emailPage.sendTestEmail(page, global.BO.EMAIL);
+      await expect(textResult).to.contains(emailPage.sendTestEmailSuccessfulMessage);
+    });
+
+    it('should check if test mail is received', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkMailBox', baseContext);
+
+      await expect(newMail.subject).to.contains(testMailSubject);
+    });
+
```

Comment:
```suggestion
```

We should not test this in this scenario, you can delete it and later we will crate the proper scenario for this.